node_frequency: 2
